reference/constraints/Traverse.rst

```diff
@@ -71,28 +105,105 @@ configure the ``Traverse`` constraint on the ``Book`` class.
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping https://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
 
-            <class name="AppBundle\Entity\Book">
-                <constraint name="Symfony\Component\Validator\Constraints\Traverse"/>
+            <class name="AppBundle\Entity\BookCollection">
+                <constraint name="Traverse"/>
             </class>
         </constraint-mapping>
 
     .. code-block:: php
 
-        // src/AppBundle/Entity/Book.php
+        // src/AppBundle/Entity/BookCollection.php
         namespace AppBundle\Entity;
 
         use Symfony\Component\Validator\Constraints as Assert;
         use Symfony\Component\Validator\Mapping\ClassMetadata;
 
-        class Book
+        class BookCollection
         {
+            // ...
+
             public static function loadValidatorMetadata(ClassMetadata $metadata)
             {
                 $metadata->addConstraint(new Assert\Traverse());
             }
         }
 
+When the object implements ``\Traversable`` (like here with its child
+``\IteratorAggregate``), its traversal strategy will implicitly be set and the
+object will be iterated over without defining the constraint.
+It's mostly useful to add it to be explicit or to disable the traversal using
+the ``traverse`` option.
+If a public getter exists to return the inner books collection like
+``getBooks(): Collection``, the :doc:`/reference/constraints/Valid` constraint
+can be used on the ``$books`` property instead.
+
 Options
 -------
 
+The ``groups`` option is not available for this constraint.
+
+
+``traverse``
+~~~~~~~~~~~~
+
+**type**: ``bool`` **default**: ``true``
+
+Instances of `\Traversable` are traversed by default, use this option to
```

Comment:
DOCtor-RST suggests to double the backslash but I would rather double the backticks